ui-library-transfer/src/App.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  AzureCommunicationTokenCredential,
+  CommunicationUserIdentifier,
+} from "@azure/communication-common";
+import {
+  CallAdapter,
+  CallComposite,
+  fromFlatCommunicationIdentifier,
+  useAzureCommunicationCallAdapter
+} from "@azure/communication-react";
+import { initializeIcons } from "@fluentui/react";
+import { useCallback, useMemo } from "react";
+import { INPUTS } from "./INPUTS";
+
+initializeIcons();
+/**
+ * Entry point of your application.
+ */
+function App(): JSX.Element {
+  // Memoize arguments to `useAzureCommunicationCallAdapter` so that
+  // a new adapter is only created when an argument changes.
+  const callAdapterArgs = useMemo(() => {
+    try {
+      const { userIdentity, userToken, callLocator, displayName } =
+        INPUTS;
+      return {
+        userId: fromFlatCommunicationIdentifier(
+          userIdentity
+        ) as CommunicationUserIdentifier,
+        credential: new AzureCommunicationTokenCredential(userToken),
+        displayName: displayName,
+        locator: callLocator
+      };
+    } catch (e) {
+      console.error(e);
+      return {};
+    }
+  }, []);
+
+  const afterCallAdapterCreate = useCallback(async (adapter: CallAdapter) => {
+    adapter.on("transferRequested", (e) => {
+      e.accept();
```

Comment:
In the quick start should we give an example on creating a simple modal for Contoso? or do we want to rely on storybook for that?

Reply:
I removed the modal example from storybook to avoid the timeout scenario.